dgl_ptm/dgl_ptm/model/initialize_model.py

```diff
@@ -186,6 +221,8 @@ def set_model_parameters(self,*,parameterFilePath=None, default=True, **kwargs):
                 else:
                     raise ValueError('default model has not been selected, but no model parameters have been supplied')
 
+        if self.restart:
```

Comment:
Why are we setting step count to restart -1? the save state should reflect the state of the system at the end of that stepn(x), so restart should intialize to that state, and then the following step would be x+1, right?